src/site/markdown/server/manage-webadmin.md

```diff
@@ -2517,14 +2517,15 @@ Deleted messages of a specific user can be restored by calling the following end
 curl -XPOST http://ip:port/deletedMessages/user/userToRestore@domain.ext
```

Comment:
How does the query looks like? Which operator/field are supported? Types for values? We need more information....